src/app/app.component.ts

```diff
@@ -80,6 +93,22 @@ export class AppComponent implements OnInit {
       });
   }
 
+  toggle(value: string) {
+    if (value === 'toggle' && !this.window.isXS) {
+      this.sideMenu = !this.sideMenu;
+      return;
+    }
+    if (!this.sideMenu) {
+      this.isPinned = value === 'enter';
+    }
+  }
+
+  toggleSideNav(sidNav: MdSidenav) {
```

Comment:
what is a sidNav?

Reply:
It is the object that we referenced for toggling